src/test/java/bio/terra/service/filedata/DrsTest.java

```diff
@@ -170,6 +180,29 @@ public void drsHackyTest() throws Exception {
     validateBQJobUserRolePresent(
         Arrays.asList(snapshotIamRoles.get(IamRole.STEWARD), snapshotIamRoles.get(IamRole.READER)));
 
+    Optional<DRSAccessMethod> drsAccessMethod =
+        drsObjectFile.getAccessMethods().stream()
+            .filter(accessMethod -> accessMethod.getType().equals(TypeEnum.GS))
+            .findFirst();
+
+    if (drsAccessMethod.isEmpty()) {
```

Comment:
You can just write this as:
`assertThat("DRS access method is present", drsAccessMethod.isPresent(), equalTo(true))`